src/ctest.ts

```diff
@@ -247,29 +247,29 @@ export class CTestDriver implements vscode.Disposable {
 
     private async getCTestArgs(driver: CMakeDriver, customizedTask: boolean = false, testPreset?: TestPreset): Promise<string[] | undefined> {
         let ctestArgs: string[];
+        const opts = driver.expansionOptions;
+        const initialArgs = await Promise.all(this.ws.config.ctestDefaultArgs.map(async (value) => expandString(value, driver.expansionOptions)));
+        const additionalArgs = await Promise.all(this.ws.config.ctestArgs.map(async (value) => expandString(value, driver.expansionOptions)));
+
+        ctestArgs = initialArgs.slice(0);
+
         if (customizedTask && testPreset) {
-            ctestArgs = ['-T', 'test'].concat(testArgs(testPreset));
```

Comment:
where are `[-T, test]` now defined? Are they still being used?

Reply:
They are still being used, however, we are falling back to allowing the user to have control over it. They are now coming from the `ctestDefaultArgs` (see line 251 in the new code). In package.json, the default values for `ctestDefaultArgs` are `-T, test, --output-on-failure`. So, instead of hardcoding this in, we are now allowing the user to modify the default arguments that are always appended to the front of the command.